tools/auto-fuzz/base_files.py

```diff
@@ -158,8 +158,11 @@ def gen_builder_1_jvm():
   RUNTIME_CLASSPATH=$RUNTIME_CLASSPATH\$this_dir/$(basename $JARFILE):
 done
 
-BUILD_CLASSPATH=$BUILD_CLASSPATH:$JAZZER_API_PATH
-RUNTIME_CLASSPATH=$RUNTIME_CLASSPATH:\$this_dir
+# Retrieve apache-common-lang3 library
```

Comment:
Could we add some more documentation what it's used for? Basically a refined version of the PR text but in the code itself.

Reply:
Done.